paper.md

```diff
@@ -227,26 +226,26 @@ For example, a workflow developed on the local machine can be run on an HPC clus
 The configuration file can be passed through the CLI interface and can be shared among different workflows.
 It can either be created by users or provided by system administrators.
 
-![Architecture of the Popper workflow engine](./figures/architecture.pdf){#fig:arch}
+![Architecture of the Popper workflow engine](./figures/architecture_med.pdf){#fig:arch}
 
 ## Continuous Integration
 
 Popper allows users to continuously validate their workflows by allowing them to export workflows as CI pipelines for different continuous integration services like Travis, Circle, Jenkins, etc.
 The tool provides a `ci` subcommand that can be used to generate CI configuration files for different CI services.
-To set up CI for a project using Popper, it is required to generate a CI configuration file, push the project to Github and enable the repository on the CI provider.
+To set up CI for a project using Popper, it is required to generate a CI configuration file, push the project to GitHub and enable the repository on the CI provider.
 Using CI with Popper workflows enhances the reproducibility guarantees as continuous validation helps to keep a check on various breaking changes like outdated dependencies, broken links, deleted Docker images, etc.
 Another benefit of using CI with Popper is that even without changes, jobs can be configured so that they run periodically (e.g. once a week), to ensure that they are in a healthy state.
 
 # Case Study {#sec:casestudy}
 
 In this section, we present three case studies demonstrating how the Popper workflow engine allows reproducing and scaling workflows in different computing environments.
-We analyzed the ML-based system benchmarking project MLPerf [@mattson2019mlperf] based on the reproducibility related issues that get frequently opened on its Github repository 
+We analyzed the ML-based system benchmarking project MLPerf [@mattson2019mlperf] based on the reproducibility related issues that get frequently opened on its GitHub repository 
```

Comment:
we can also add a link to the repository

Reply:
also, we can reiterate how Popper is solving this in the Results section, as this is a central point.